spec/features/session_spec.cr

```diff
@@ -10,9 +10,9 @@ module Selenium::Command
       HTML
 
       begin
-        driver1, args1 = Selenium::TestDriverFactory.build(ENV["SELENIUM_BROWSER"]? || "chrome")
+        driver1, args1 = Selenium::TestDriverFactory.build(ENV["SELENIUM_BROWSER"]? || "chrome", 9222)
         session1 = driver1.create_session(args: args1)
-        driver2, args2 = Selenium::TestDriverFactory.build(ENV["SELENIUM_BROWSER"]? || "chrome")
+        driver2, args2 = Selenium::TestDriverFactory.build(ENV["SELENIUM_BROWSER"]? || "chrome", 9223)
```

Comment:
The latest Chromedriver seems to not auto increment the debugging port. If these are both set to the same port, then the sessions are duplicated which means visiting a page on one will return that the other also visited that page.